src/FileExtensionManager.cpp

```diff
@@ -0,0 +1,108 @@
+#include "FileExtensionManager.h"
+#include "ui_FileExtensionManager.h"
+
+#include "FileDialog.h"
```

Comment:
This include doesn't seem to be needed.

Reply:
Yep, it's there from a previous attempt ;)